compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -828,6 +828,32 @@ class PowTestGraph final : public SimpleTestGraph
   luci::CircleConst *_const = nullptr;
 };
 
+class ResizeBilinearTestGraph final : public luci::test::TestIOGraph
+{
+public:
+  void init(void)
+  {
+    TestIOGraph::init({1, 4, 4, 1}, {1, 8, 8, 1});
+
+    _size = create_const<Type::S32, int32_t>(g(), {2}, {8, 8});
+    _resizebilinear = g()->nodes()->create<luci::CircleResizeBilinear>();
+    {
+      _resizebilinear->input(input());
+      _resizebilinear->size(_size);
+    }
+    output()->from(_resizebilinear);
+
+    set_minmax_to_non_const(g(), -1, 1);
+  }
+
+  // Avoid overriding TestIOGraph::input()
+  loco::Node *input_rbl() { return _resizebilinear->input(); }
```

Comment:
I cann't see where this is used. Will it be used in another PR?

Reply:
@seanshpark Sorry, it's not yet rebased.. I cannot fetch from the upstream for now

Reply:
Oh, the answer to what you point out is: no, it isn't used. I will remove it. 😄